mavis/python/mavis_tacplus_keycloak.py

```
@@ -0,0 +1,220 @@
+	post_data = {
+		'grant_type': 'password',
+		'client_id': KEYCLOAK_CLIENT_ID,
+		'username': D.user,
+		'password': D.password,
+	}
+	if KEYCLOAK_CLIENT_SECRET:
+		post_data['client_secret'] = KEYCLOAK_CLIENT_SECRET
+
+	try:
+		resp = http.post(TOKEN_URL, data=post_data, timeout=KEYCLOAK_TIMEOUT)
+	except Exception as e:
```

Ruff (0.14.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)